bftengine/src/preprocessor/messages/PreProcessReplyMsg.cpp

```diff
@@ -19,7 +19,7 @@ using namespace std;
 using namespace concord::util;
 using namespace bftEngine;
 
-uint16_t PreProcessReplyMsg::maxReplyMsgSize_ = 512;  // Actually, it is sizeof(PreProcessReplyMsgHeader) = 50 + 256
+uint16_t PreProcessReplyMsg::maxReplyMsgSize_ = 1024;  // Actually, it is sizeof(Header) = 58 + 256 + cid.size()
```

Comment:
The comment is not very clear (because of the mix of constants end expressions, I think). Can we say instead:

`Actually we should have maxReplyMsgSize_ >= size(header) + size(signature) + size(cid) >= 58 + 256 + ? bytes`

PS: I'm not sure what the size of the CID is :)

Reply:
Me either, but we need to limit the message size somehow.

Reply:
I see. My main suggestion is to clarify the comment - I'm not questioning the need of a `maxReplyMsgSize_`.

Reply:
OK

Reply:
Nice, thanks!